mean&Median/mean_median.go

```diff
@@ -0,0 +1,53 @@
+package main
+
+import (
+	"log"
+	"sort"
+	"sync"
+	"time"
+)
+
+func num(v int, wg *sync.WaitGroup, ch1 chan int) {
+	time.Sleep(1 * time.Second)
+	ch1 <- v
+	wg.Done()
+}
+
```

Comment:
@Prashanth6361 please try solving this without sort.Ints (internal library)

Reply:
[]int{1,2,3,4}  2.5 should be the median right ? can you check if gives the same result.

Reply:
Since, I have taken median as int, the output is displaying as 2. But,  if we use median as float we will get the output as 2.5